lib/active_merchant/billing/gateways/authorize_net.rb

```diff
@@ -176,13 +176,40 @@ def credit(amount, payment, options = {})
         end
       end
 
-      def verify(credit_card, options = {})
+      def verify(payment_method, options = {})
+        amount = define_amount_for_verify(payment_method, options)
+        return amount if amount.class == Response
```

Comment:
FYI this can be 
```suggestion
        return amount if amount.is_a?(Response)
```

Reply:
solved! please check again!

Reply:
@jherreraa Not sure if you need to push again, but this doesn't seem to have been changed - ping me again when the PR is up to date